src/invoice2data/extract/parsers/regex.py

```diff
@@ -46,6 +46,14 @@ def parse(template, field, settings, content, legacy=False):
     if "group" in settings:
         if settings["group"] == "sum":
             result = sum(result)
+        elif settings["group"] == "min":
```

Comment:
Very interesting what your doing there.
Would this also work for date fields??

Currently when, a reges matches multiple date fields, and dict error occurs.
Would be interesting to prevent these dict errors.
e.g. Being able to return the oldest date as invoice date. Latest date as the due date.